build/vega-lite-schema.json

```diff
@@ -2816,6 +2826,64 @@
       },
       "type": "object"
     },
+    "LogicalAnd<string>": {
```

Comment:
`<string>` wouldn't be friendly for generating commands in Altair. 

Maybe add rename commands for `LogicalAnd/Or/Not/Operand<string>` to `scripts/rename-schema.sh`?